workitem/link/link_repository.go

```diff
@@ -91,7 +93,7 @@ func (r *GormWorkItemLinkRepository) ValidateCorrectSourceAndTargetType(ctx cont
 
 // Create creates a new work item link in the repository.
 // Returns BadParameterError, ConversionError or InternalError
-func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targetID uint64, linkTypeID satoriuuid.UUID) (*app.WorkItemLinkSingle, error) {
+func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targetID uint64, linkTypeID satoriuuid.UUID, creatorID satoriuuid.UUID) (*app.WorkItemLinkSingle, error) {
```

Comment:
@aslakknutsen I don't disagree fully but let's say you want to store the IP address of the person that made the edit. That isn't an API requirement but a storage requirement. Where the IP address comes from is not relevant when formulating this requirement. My guess would be that the context has something like this already available and if not I would still see the context or the request as the source of this information.